### ISSUE TITLE
Can' install portmaster any more

### ISSUE THREAD

**Issue by Flashwalker** [wimpysworld/deb-get#888]
```bash
$ curl -sL https://raw.githubusercontent.com/wimpysworld/deb-get/main/deb-get | sudo -E bash -s install deb-get
$ deb-get clean
$ deb-get update
$ deb-get install portmaster
  [!] ERROR! Missing required information of website package portmaster:
URL=https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb
VERSION_PUBLISHED=
```

### Expected behavior

Install portmaster
### Actual behavior

Can' install portmaster any more
### Steps to reproduce

Describe the exact steps required to reproduce the issue.

#### /etc/os-release

```bash
NAME="Feren OS"
VERSION="2023.04"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Feren OS"
VERSION_ID="20.04"
HOME_URL="https://ferenos.weebly.com/"
SUPPORT_URL="https://ferenos.weebly.com/feren-os-on-social-media.html"
BUG_REPORT_URL="https://github.com/feren-OS/Bug-Reporting-Center/issues/new"
PRIVACY_POLICY_URL="https://ferenos.weebly.com/privacy-policy.html"
VERSION_CODENAME=focal
FEREN_CODENAME=nihonium
```

#### /etc/lsb-release

```bash
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Feren OS"
```

#### deb-get version

```bash
0.3.6
```
### deb-get update

<details>

<summary>### deb-get update</summary>

```bash
  [+] Updating /etc/deb-get/01-main
/etc/deb-get/01-main.repo                      100%[==================================================================================================>]   2,95K  --.-KB/s    за 0s      
1password                                      100%[==================================================================================================>]     306  --.-KB/s    за 0s      
agena                                          100%[==================================================================================================>]   1,01K  --.-KB/s    за 0s      
alduin                                         100%[==================================================================================================>]     463  --.-KB/s    за 0s      
android-messages-desktop                       100%[==================================================================================================>]     450  --.-KB/s    за 0s      
antimicrox                                     100%[==================================================================================================>]     439  --.-KB/s    за 0s      
anydesk                                        100%[==================================================================================================>]     278  --.-KB/s    за 0s      
appimagelauncher                               100%[==================================================================================================>]     636  --.-KB/s    за 0s      
atom                                           100%[==================================================================================================>]     270  --.-KB/s    за 0s      
atomic                                         100%[==================================================================================================>]     443  --.-KB/s    за 0s      
audio-recorder                                 100%[==================================================================================================>]     299  --.-KB/s    за 0s      
authme                                         100%[==================================================================================================>]     395  --.-KB/s    за 0s      
azure-cli                                      100%[==================================================================================================>]     453  --.-KB/s    за 0s      
azuredatastudio                                100%[==================================================================================================>]     502  --.-KB/s    за 0s      
balena-etcher                                  100%[==================================================================================================>]     377  --.-KB/s    за 0s      
bat                                            100%[==================================================================================================>]     402  --.-KB/s    за 0s      
battery-monitor                                100%[==================================================================================================>]     537  --.-KB/s    за 0s      
beersmith3                                     100%[==================================================================================================>]     824  --.-KB/s    за 0s      
bitwarden                                      100%[==================================================================================================>]     371  --.-KB/s    за 0s      
bitwig-studio                                  100%[==================================================================================================>]     388  --.-KB/s    за 0s      
blanket                                        100%[==================================================================================================>]     253  --.-KB/s    за 0s      
blockbench                                     100%[==================================================================================================>]     365  --.-KB/s    за 0s      
bluejeans-v2                                   100%[==================================================================================================>]     445  --.-KB/s    за 0s      
bottom                                         100%[==================================================================================================>]     405  --.-KB/s    за 0s      
brave-browser                                  100%[==================================================================================================>]     343  --.-KB/s    за 0s      
brisqi                                         100%[==================================================================================================>]     371  --.-KB/s    за 0s      
caddy                                          100%[==================================================================================================>]     421  --.-KB/s    за 0s      
caprine                                        100%[==================================================================================================>]     371  --.-KB/s    за 0s      
cawbird                                        100%[==================================================================================================>]     617  --.-KB/s    за 0s      
chronograf                                     100%[==================================================================================================>]     360  --.-KB/s    за 0s      
code                                           100%[==================================================================================================>]     306  --.-KB/s    за 0s      
codium                                         100%[==================================================================================================>]     436  --.-KB/s    за 0s      
com.github.tkashkin.gamehub                    100%[==================================================================================================>]     412  --.-KB/s    за 0s      
copyq                                          100%[==================================================================================================>]     167  --.-KB/s    за 0s      
corectrl                                       100%[==================================================================================================>]     246  --.-KB/s    за 0s      
cosign                                         100%[==================================================================================================>]     438  --.-KB/s    за 0s      
croc                                           100%[==================================================================================================>]     781  --.-KB/s    за 0s      
cronopete                                      100%[==================================================================================================>]     551  --.-KB/s    за 0s      
crossover                                      100%[==================================================================================================>]     554  --.-KB/s    за 0s      
crow-translate                                 100%[==================================================================================================>]     510  --.-KB/s    за 0s      
cryptomator                                    100%[==================================================================================================>]     198  --.-KB/s    за 0s      
dbeaver-ce                                     100%[==================================================================================================>]     400  --.-KB/s    за 0s      
deadbeef                                       100%[==================================================================================================>]     391  --.-KB/s    за 0s      
deadbeef-static                                100%[==================================================================================================>]     391  --.-KB/s    за 0s      
deborah                                        100%[==================================================================================================>]     433  --.-KB/s    за 0s      
deltachat-desktop                              100%[==================================================================================================>]     349  --.-KB/s    за 0s      
deskcut                                        100%[==================================================================================================>]     437  --.-KB/s    за 0s      
discord                                        100%[==================================================================================================>]     326  --.-KB/s    за 0s      
dive                                           100%[==================================================================================================>]     416  --.-KB/s    за 0s      
docker-ce                                      100%[==================================================================================================>]     419  --.-KB/s    за 0s      
docker-desktop                                 100%[==================================================================================================>]     426  --.-KB/s    за 0s      
draw.io                                        100%[==================================================================================================>]     403  --.-KB/s    за 0s      
dropbox                                        100%[==================================================================================================>]     623  --.-KB/s    за 0s      
du-dust                                        100%[==================================================================================================>]     387  --.-KB/s    за 0s      
duf                                            100%[==================================================================================================>]     584  --.-KB/s    за 0s      
dustracing2d                                   100%[==================================================================================================>]     461  --.-KB/s    за 0s      
element-desktop                                100%[==================================================================================================>]     302  --.-KB/s    за 0s      
enpass                                         100%[==================================================================================================>]     245  --.-KB/s    за 0s      
exifcleaner                                    100%[==================================================================================================>]     411  --.-KB/s    за 0s      
exodus                                         100%[==================================================================================================>]     406  --.-KB/s    за 0s      
expressvpn                                     100%[==================================================================================================>]     488  --.-KB/s    за 0s      
fastfetch                                      100%[==================================================================================================>]     526  --.-KB/s    за 0s      
fd                                             100%[==================================================================================================>]     429  --.-KB/s    за 0s      
ferdium                                        100%[==================================================================================================>]     482  --.-KB/s    за 0s      
figma-linux                                    100%[==================================================================================================>]     433  --.-KB/s    за 0s      
filezilla-theme-papirus                        100%[==================================================================================================>]     256  --.-KB/s    за 0s      
firefox-esr                                    100%[==================================================================================================>]     160  --.-KB/s    за 0s      
flameshot                                      100%[==================================================================================================>]     734  --.-KB/s    за 0s      
foliate                                        100%[==================================================================================================>]     172  --.-KB/s    за 0s      
franz                                          100%[==================================================================================================>]     400  --.-KB/s    за 0s      
frostwire                                      100%[==================================================================================================>]     448  --.-KB/s    за 0s      
fsearch                                        100%[==================================================================================================>]     162  --.-KB/s    за 0s      
gcm                                            100%[==================================================================================================>]     523  --.-KB/s    за 0s      
geforcenow-electron                            100%[==================================================================================================>]     396  --.-KB/s    за 0s      
gh                                             100%[==================================================================================================>]     418  --.-KB/s    за 0s      
git-delta                                      100%[==================================================================================================>]     458  --.-KB/s    за 0s      
github-desktop                                 100%[==================================================================================================>]     387  --.-KB/s    за 0s      
gitkraken                                      100%[==================================================================================================>]     420  --.-KB/s    за 0s      
gitter                                         100%[==================================================================================================>]     445  --.-KB/s    за 0s      
glow                                           100%[==================================================================================================>]     397  --.-KB/s    за 0s      
google-chat-electron                           100%[==================================================================================================>]     412  --.-KB/s    за 0s      
google-chrome-stable                           100%[==================================================================================================>]     526  --.-KB/s    за 0s      
google-cloud-cli                               100%[==================================================================================================>]     434  --.-KB/s    за 0s      
google-earth-pro-stable                        100%[==================================================================================================>]     385  --.-KB/s    за 0s      
gpu-viewer                                     100%[==================================================================================================>]     192  --.-KB/s    за 0s      
grype                                          100%[==================================================================================================>]     416  --.-KB/s    за 0s      
headset                                        100%[==================================================================================================>]     300  --.-KB/s    за 0s      
helio-workstation                              100%[==================================================================================================>]     310  --.-KB/s    за 0s      
heroic                                         100%[==================================================================================================>]     427  --.-KB/s    за 0s      
homeassistant-supervised                       100%[==================================================================================================>]     507  --.-KB/s    за 0s      
hugo                                           100%[==================================================================================================>]     410  --.-KB/s    за 0s      
hyper                                          100%[==================================================================================================>]     400  --.-KB/s    за 0s      
i3                                             100%[==================================================================================================>]     367  --.-KB/s    за 0s      
igdm                                           100%[==================================================================================================>]     406  --.-KB/s    за 0s      
igdm-pro                                       100%[==================================================================================================>]     410  --.-KB/s    за 0s      
indicator-sound-switcher                       100%[==================================================================================================>]     214  --.-KB/s    за 0s      
influxdb                                       100%[==================================================================================================>]     354  --.-KB/s    за 0s      
influxdb2                                      100%[==================================================================================================>]     349  --.-KB/s    за 0s      
influxdb2-cli                                  100%[==================================================================================================>]     330  --.-KB/s    за 0s      
insomnia                                       100%[==================================================================================================>]     390  --.-KB/s    за 0s      
insync                                         100%[==================================================================================================>]     277  --.-KB/s    за 0s      
ipfs-desktop                                   100%[==================================================================================================>]     479  --.-KB/s    за 0s      
irccloud-desktop                               100%[==================================================================================================>]     368  --.-KB/s    за 0s      
iriunwebcam                                    100%[==================================================================================================>]     332  --.-KB/s    за 0s      
jabref                                         100%[==================================================================================================>]     376  --.-KB/s    за 0s      
jami                                           100%[==================================================================================================>]     233  --.-KB/s    за 0s      
jellyfin                                       100%[==================================================================================================>]     330  --.-KB/s    за 0s      
jellyfin-media-player                          100%[==================================================================================================>]     584  --.-KB/s    за 0s      
jexiftoolgui                                   100%[==================================================================================================>]     486  --.-KB/s    за 0s      
jupyterlab-desktop                             100%[==================================================================================================>]     410  --.-KB/s    за 0s      
kapacitor                                      100%[==================================================================================================>]     353  --.-KB/s    за 0s      
kdiskmark                                      100%[==================================================================================================>]     180  --.-KB/s    за 0s      
keepassxc                                      100%[==================================================================================================>]     139  --.-KB/s    за 0s      
keybase                                        100%[==================================================================================================>]     288  --.-KB/s    за 0s      
kopia-ui                                       100%[==================================================================================================>]     349  --.-KB/s    за 0s      
koreader                                       100%[==================================================================================================>]     421  --.-KB/s    за 0s      
ksnip                                          100%[==================================================================================================>]     372  --.-KB/s    за 0s      
kwaterfoxhelper                                100%[==================================================================================================>]     791  --.-KB/s    за 0s      
lens                                           100%[==================================================================================================>]     276  --.-KB/s    за 0s      
libreoffice                                    100%[==================================================================================================>]     245  --.-KB/s    за 0s      
librewolf                                      100%[==================================================================================================>]     362  --.-KB/s    за 0s      
limoji                                         100%[==================================================================================================>]     403  --.-KB/s    за 0s      
lsd                                            100%[==================================================================================================>]     404  --.-KB/s    за 0s      
ludo                                           100%[==================================================================================================>]     408  --.-KB/s    за 0s      
lutris                                         100%[==================================================================================================>]     365  --.-KB/s    за 0s      
mailspring                                     100%[==================================================================================================>]     404  --.-KB/s    за 0s      
makedeb                                        100%[==================================================================================================>]     440  --.-KB/s    за 0s      
master-pdf-editor-5                            100%[==================================================================================================>]     587  --.-KB/s    за 0s      
mattermost-desktop                             100%[==================================================================================================>]     527  --.-KB/s    за 0s      
media-downloader                               100%[==================================================================================================>]     714  --.-KB/s    за 0s      
mediathekview                                  100%[==================================================================================================>]     577  --.-KB/s    за 0s      
mergerfs                                       100%[==================================================================================================>]     709  --.-KB/s    за 0s      
micro                                          100%[==================================================================================================>]     383  --.-KB/s    за 0s      
microsoft-edge-stable                          100%[==================================================================================================>]     365  --.-KB/s    за 0s      
min                                            100%[==================================================================================================>]     587  --.-KB/s    за 0s      
minigalaxy                                     100%[==================================================================================================>]     403  --.-KB/s    за 0s      
minikube                                       100%[==================================================================================================>]     479  --.-KB/s    за 0s      
motrix                                         100%[==================================================================================================>]     342  --.-KB/s    за 0s      
mpdevil                                        100%[==================================================================================================>]     358  --.-KB/s    за 0s      
ms-office-electron                             100%[==================================================================================================>]     433  --.-KB/s    за 0s      
mullvad-vpn                                    100%[==================================================================================================>]     434  --.-KB/s    за 0s      
nala                                           100%[==================================================================================================>]     399  --.-KB/s    за 0s      
nala-legacy                                    100%[==================================================================================================>]     404  --.-KB/s    за 0,001s  
neo4j                                          100%[==================================================================================================>]     245  --.-KB/s    за 0s      
neovim                                         100%[==================================================================================================>]     464  --.-KB/s    за 0s      
nextcloud-desktop                              100%[==================================================================================================>]     181  --.-KB/s    за 0s      
nomad                                          100%[==================================================================================================>]     360  --.-KB/s    за 0s      
nordvpn                                        100%[==================================================================================================>]     534  --.-KB/s    за 0s      
notable                                        100%[==================================================================================================>]     364  --.-KB/s    за 0s      
nuclear                                        100%[==================================================================================================>]     463  --.-KB/s    за 0s      
obs-cli                                        100%[==================================================================================================>]     421  --.-KB/s    за 0s      
obsidian                                       100%[==================================================================================================>]     409  --.-KB/s    за 0s      
obs-studio                                     100%[==================================================================================================>]     204  --.-KB/s    за 0s      
ocenaudio                                      100%[==================================================================================================>]     403  --.-KB/s    за 0s      
onedriver                                      100%[==================================================================================================>]     618  --.-KB/s    за 0s      
onlyoffice-desktopeditors                      100%[==================================================================================================>]     444  --.-KB/s    за 0s      
openaudible                                    100%[==================================================================================================>]     442  --.-KB/s    за 0s      
openrazer-meta                                 100%[==================================================================================================>]     195  --.-KB/s    за 0s      
openrgb                                        100%[==================================================================================================>]     887  --.-KB/s    за 0s      
opera-stable                                   100%[==================================================================================================>]     236  --.-KB/s    за 0s      
os-agent                                       100%[==================================================================================================>]     581  --.-KB/s    за 0s      
p3x-onenote                                    100%[==================================================================================================>]     379  --.-KB/s    за 0s      
pandoc                                         100%[==================================================================================================>]     369  --.-KB/s    за 0s      
papirus-icon-theme                             100%[==================================================================================================>]     285  --.-KB/s    за 0s      
parsec                                         100%[==================================================================================================>]     190  --.-KB/s    за 0s      
pdfsam-basic                                   100%[==================================================================================================>]     551  --.-KB/s    за 0s      
pdfsam-visual                                  100%[==================================================================================================>]     643  --.-KB/s    за 0s      
peazip                                         100%[==================================================================================================>]     389  --.-KB/s    за 0s      
picocrypt                                      100%[==================================================================================================>]     402  --.-KB/s    за 0s      
plexmediaserver                                100%[==================================================================================================>]     258  --.-KB/s    за 0s      
plow                                           100%[==================================================================================================>]     448  --.-KB/s    за 0s      
polychromatic                                  100%[==================================================================================================>]     203  --.-KB/s    за 0s      
pomatez                                        100%[==================================================================================================>]     378  --.-KB/s    за 0s      
portmaster                                     100%[==================================================================================================>]     516  --.-KB/s    за 0s      
powershell                                     100%[==================================================================================================>]     466  --.-KB/s    за 0s      
protonmail-bridge                              100%[==================================================================================================>]     456  --.-KB/s    за 0s      
protonvpn                                      100%[==================================================================================================>]     282  --.-KB/s    за 0s      
publii                                         100%[==================================================================================================>]     479  --.-KB/s    за 0s      
pulsar                                         100%[==================================================================================================>]     402  --.-KB/s    за 0s      
pulse-sms                                      100%[==================================================================================================>]     374  --.-KB/s    за 0s      
qownnotes                                      100%[==================================================================================================>]     197  --.-KB/s    за 0s      
quarto                                         100%[==================================================================================================>]     416  --.-KB/s    за 0s      
quickemu                                       100%[==================================================================================================>]     214  --.-KB/s    за 0s      
quickgui                                       100%[==================================================================================================>]     164  --.-KB/s    за 0s      
quickobs                                       100%[==================================================================================================>]     477  --.-KB/s    за 0s      
rambox                                         100%[==================================================================================================>]     347  --.-KB/s    за 0s      
rclone                                         100%[==================================================================================================>]     603  --.-KB/s    за 0s      
resilio-sync                                   100%[==================================================================================================>]     323  --.-KB/s    за 0s      
retroarch                                      100%[==================================================================================================>]     178  --.-KB/s    за 0s      
rocketchat                                     100%[==================================================================================================>]     384  --.-KB/s    за 0s      
rpi-imager                                     100%[==================================================================================================>]     384  --.-KB/s    за 0s      
rstudio                                        100%[==================================================================================================>]     672  --.-KB/s    за 0s      
rstudio-server                                 100%[==================================================================================================>]     689  --.-KB/s    за 0s      
rustdesk                                       100%[==================================================================================================>]     655  --.-KB/s    за 0s      
sejda-desktop                                  100%[==================================================================================================>]     362  --.-KB/s    за 0s      
sengi                                          100%[==================================================================================================>]     399  --.-KB/s    за 0s      
shutter-encoder                                100%[==================================================================================================>]     432  --.-KB/s    за 0s      
signal-desktop                                 100%[==================================================================================================>]     277  --.-KB/s    за 0s      
simplenote                                     100%[==================================================================================================>]     574  --.-KB/s    за 0s      
skypeforlinux                                  100%[==================================================================================================>]     289  --.-KB/s    за 0s      
slack-desktop                                  100%[==================================================================================================>]     284  --.-KB/s    за 0s      
sleek                                          100%[==================================================================================================>]     402  --.-KB/s    за 0s      
smartgit                                       100%[==================================================================================================>]     436  --.-KB/s    за 0s      
sniffnet                                       100%[==================================================================================================>]     403  --.-KB/s    за 0s      
softmaker-office-2021                          100%[==================================================================================================>]     311  --.-KB/s    за 0s      
soundux                                        100%[==================================================================================================>]     511  --.-KB/s    за 0s      
spotify-client                                 100%[==================================================================================================>]     247  --.-KB/s    за 0s      
standard-notes                                 100%[==================================================================================================>]     573  --.-KB/s    за 0s      
strawberry                                     100%[==================================================================================================>]     222  --.-KB/s    за 0s      
stremio                                        100%[==================================================================================================>]     411  --.-KB/s    за 0s      
sublime-merge                                  100%[==================================================================================================>]     268  --.-KB/s    за 0s      
sublime-text                                   100%[==================================================================================================>]     237  --.-KB/s    за 0s      
sunshine                                       100%[==================================================================================================>]     670  --.-KB/s    за 0s      
surfshark                                      100%[==================================================================================================>]     507  --.-KB/s    за 0s      
syft                                           100%[==================================================================================================>]     455  --.-KB/s    за 0s      
syncthing                                      100%[==================================================================================================>]     229  --.-KB/s    за 0s      
system-monitoring-center                       100%[==================================================================================================>]     509  --.-KB/s    за 0s      
tabby-terminal                                 100%[==================================================================================================>]     378  --.-KB/s    за 0s      
tailscale                                      100%[==================================================================================================>]     382  --.-KB/s    за 0s      
tarsnap                                        100%[==================================================================================================>]     331  --.-KB/s    за 0s      
teams                                          100%[==================================================================================================>]     317  --.-KB/s    за 0s      
teams-for-linux                                100%[==================================================================================================>]     648  --.-KB/s    за 0s      
teamviewer                                     100%[==================================================================================================>]     261  --.-KB/s    за 0s      
telegraf                                       100%[==================================================================================================>]     340  --.-KB/s    за 0s      
terraform                                      100%[==================================================================================================>]     309  --.-KB/s    за 0s      
texworks                                       100%[==================================================================================================>]     149  --.-KB/s    за 0s      
tezos-baking                                   100%[==================================================================================================>]     223  --.-KB/s    за 0s      
thinlinc-client                                100%[==================================================================================================>]     721  --.-KB/s    за 0s      
tidal-hifi                                     100%[==================================================================================================>]     446  --.-KB/s    за 0s      
tixati                                         100%[==================================================================================================>]     339  --.-KB/s    за 0s      
tonelib-bassdrive                              100%[==================================================================================================>]     535  --.-KB/s    за 0s      
tonelib-gfx                                    100%[==================================================================================================>]     537  --.-KB/s    за 0s      
tonelib-jam                                    100%[==================================================================================================>]     562  --.-KB/s    за 0s      
tonelib-metal                                  100%[==================================================================================================>]     512  --.-KB/s    за 0s      
tonelib-noisereducer                           100%[==================================================================================================>]     515  --.-KB/s    за 0s      
tonelib-tubewarmth                             100%[==================================================================================================>]     520  --.-KB/s    за 0s      
tonelib-zoom                                   100%[==================================================================================================>]     526  --.-KB/s    за 0s      
tribler                                        100%[==================================================================================================>]     358  --.-KB/s    за 0s      
trilium                                        100%[==================================================================================================>]     460  --.-KB/s    за 0s      
trivy                                          100%[==================================================================================================>]     814  --.-KB/s    за 0s      
typora                                         100%[==================================================================================================>]     202  --.-KB/s    за 0s      
ubuntu-make                                    100%[==================================================================================================>]     180  --.-KB/s    за 0s      
ulauncher                                      100%[==================================================================================================>]     138  --.-KB/s    за 0s      
ungoogled-chromium                             100%[==================================================================================================>]     709  --.-KB/s    за 0s      
urserver                                       100%[==================================================================================================>]     334  --.-KB/s    за 0s      
usbimager                                      100%[==================================================================================================>]     472  --.-KB/s    за 0s      
vagrant                                        100%[==================================================================================================>]     351  --.-KB/s    за 0s      
veracrypt                                      100%[==================================================================================================>]     202  --.-KB/s    за 0s      
virtualbox-6.1                                 100%[==================================================================================================>]     463  --.-KB/s    за 0s      
virtualbox-7.0                                 100%[==================================================================================================>]     455  --.-KB/s    за 0s      
vivaldi-stable                                 100%[==================================================================================================>]     350  --.-KB/s    за 0s      
vuescan                                        100%[==================================================================================================>]     740  --.-KB/s    за 0s      
waterfox-g-kde                                 100%[==================================================================================================>]     771  --.-KB/s    за 0s      
wavebox                                        100%[==================================================================================================>]     301  --.-KB/s    за 0s      
waydroid                                       100%[==================================================================================================>]     483  --.-KB/s    за 0s      
webex                                          100%[==================================================================================================>]     536  --.-KB/s    за 0s      
weechat                                        100%[==================================================================================================>]     280  --.-KB/s    за 0s      
weylus                                         100%[==================================================================================================>]     418  --.-KB/s    за 0s      
whalebird                                      100%[==================================================================================================>]     405  --.-KB/s    за 0s      
whatsapp-for-linux                             100%[==================================================================================================>]     421  --.-KB/s    за 0s      
winehq-devel                                   100%[==================================================================================================>]     386  --.-KB/s    за 0s      
winehq-stable                                  100%[==================================================================================================>]     387  --.-KB/s    за 0s      
winehq-staging                                 100%[==================================================================================================>]     388  --.-KB/s    за 0s      
wire-desktop                                   100%[==================================================================================================>]     252  --.-KB/s    за 0s      
wraith-master                                  100%[==================================================================================================>]     431  --.-KB/s    за 0s      
xemu                                           100%[==================================================================================================>]     271  --.-KB/s    за 0s      
youtube-music                                  100%[==================================================================================================>]     468  --.-KB/s    за 0s      
yq                                             100%[==================================================================================================>]     158  --.-KB/s    за 0s      
zenith                                         100%[==================================================================================================>]     392  --.-KB/s    за 0s      
zettlr                                         100%[==================================================================================================>]     403  --.-KB/s    за 0s      
zoom                                           100%[==================================================================================================>]     300  --.-KB/s    за 0s      
zotero                                         100%[==================================================================================================>]     363  --.-KB/s    за 0s
```

</details>

**Comment by philclifford** [comment]
You have an old version of `deb-get` there (reporting 0.3.6) which should not be the case if you did the `curl` self-install.  With the current 0.3.9 release and the current HEAD of main there is no issue with portmaster and the version reports 0.3.9.

```
$ deb-get version
0.3.9
$ deb-get show portmaster 
  [...]
Portmaster
  Package:	portmaster
  Repository:	01-main
  Updater:	deb-get
  Installed:	No
  Published:	1.4.5
  Architecture:	amd64
  Download:	https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb
  Website:	https://safing.io/portmaster/
  Summary:	Portmaster is a free and open-source application that puts you back in charge over all your computer's network connections.
```
Do you have an old copy higher in your path ? ( check `command -v deb-get ` )
It could also have been a transient upstream issue while they were adding a pre-release yesterday, in which case it should be magically working again.

**Comment by Flashwalker** [comment]
thanks, I fixed it.
